### PR TITLE
[WEB-5402]: Create a test to test routes (profile service)

### PR DESCRIPTION
**Pull request by aosmakserato**
Added an abstract test class `AbstractRouteTest`

### CODE REVIEW

File: src/Test/AbstractRouteTest.php [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+namespace Serato\SwsApp\Test;
+
+use Slim\Container;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class AbstractRouteTest
+ * @package Serato\SwsApp\Test
+ */
+abstract class AbstractRouteTest extends TestCase
```
Comment (aosmakserato) [inline src/Test/AbstractRouteTest.php:11]: abstract class to be extended by all the routing tests in our services

File: src/Test/AbstractRouteTest.php [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        // check whether the controller was registered
+        $controller = $this->getContainer()->get($expectedClassName);
+        $this->assertNotEmpty($controller);
+    }
+
+    /**
+     * @param string $callableFullNamespace
+     * @return string
+     */
+    protected function getClassNameFromString(string $callableFullNamespace): string
```
Comment (mshahidi) [inline src/Test/AbstractRouteTest.php:127]: This class can be private too right?

Comment (aosmakserato) [reply src/Test/AbstractRouteTest.php:127]: this method you mean? Yes, we can make it private

File: src/Test/AbstractRouteTest.php [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    {
+        $explodedCallableFullNamespace = explode('\\', $callableFullNamespace);
+        return end($explodedCallableFullNamespace);
```
Comment (GuillaumeBrousse) [inline src/Test/AbstractRouteTest.php:130]: you can do a tiny faster hack : 
`return explode('\\', static::class)[0];`

**Review by GuillaumeBrousse — APPROVED** [review]
Looks good to me nice work !
NB same as Mehdi can put the bottom method as private

File: src/Test/AbstractRouteTest.php (hunk 2 above)
Comment (mshahidi) [reply src/Test/AbstractRouteTest.php:127]: Sorry I meant method 😆

**Review by mshahidi — APPROVED** [review]

**Pull request merged**